modules/standard/BigInteger.chpl

```diff
@@ -104,6 +104,17 @@ cannot be represented using the GMP scalar type.
 The checks are controlled by the compiler options ``--[no-]cast-checks``,
 ``--fast``, etc.
 
+Casting from ``bigint`` to ``integral`` and ``real`` types is also
+supported.  Values that are too large for the resultant type are
+truncated.  GMP primitives are used to first cast to platform-specific C
+types, which are then cast to Chapel types.  As a result, casting to
+64-bit types on 32-bit platforms may result in additional truncation.
+Additionally, casting a negative ``bigint`` to a ``uint`` will result in
+the absolute value truncated to fit within the type.::
+
+  var a = new bigint(-1);
+  writeln(a:uint);        // prints "1"
```

Comment:
It occurs to me that we could make this print out the same as (-1):uint by doing some bit-complementing for negative numbers. It's unclear to me if this is better though so I might like a 3rd opinion.